### DISCUSSION TITLE
Set the email address for an issue

### DISCUSSION THREAD

**Discussion by robbycbennett** [community/community#74926] [Projects and Issues]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

I have a GitHub account that I use for personal projects and work. When I create an issue at work, I want that updates on that issue to go to my work email, and likewise for personal projects.

We should be able to set the preferred email per-repo and per-issue.

The interface where we could have the preferred email per issue:
![image](https://github.com/community/community/assets/22121365/4e0eb9ba-0f82-4cd9-92a1-2c082456f4de)
![image](https://github.com/community/community/assets/22121365/701eb5f5-e80a-4598-8a94-beecb55b3175)

The interface where we could have the preferred email for issues per repo:
![image](https://github.com/community/community/assets/22121365/48f604a9-0d4d-4039-ad1b-d7f495bc70e6)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by klondikemarlen** [comment]
I have a similar issue. My GitHub account is tied to my personal email, but I use it for work as well.
I'd like to propose the more general option of routing email on a per-repository or per-organization basis.

e.g
- Any correspondence against https://github.com/icefoganalytics would route to my work email.
- Any correspondence against any other namespace - for example https://github.com/klondikemarlen/patch-github-new-pull-request-redirect - would go to my personal email.

**Reply by robbycbennett** [reply]
I like that idea even more. Per-repository or per-organization makes more sense than per-issue.

**Reply by deniseharrison** [reply]
That’s a great idea! It would be super helpful to set preferred emails for each repo or issue. I’ve had similar situations managing notifications for my [Cold Email Agency](https://thecoldemailagency.co.uk/)—having updates sent to the right email would streamline everything. It’d save time and keep work and personal projects organized!
